applications/nrf_desktop/src/modules/config.c

```diff
@@ -142,6 +149,49 @@ static void update_config(const u8_t config_id, u8_t *data, size_t size)
 	}
 }
 
+static void settings_load_thread(void)
+{
+	LOG_INF("Settings load thread started");
+
+	int err = 0;
+	
+	err = settings_load();
+	if (err) {
+		LOG_ERR("Cannot load settings");
+	}
+	LOG_INF("Settings loaded");
+
+	if (err) {
+		module_set_state(MODULE_STATE_ERROR);
+	} else {
+		module_set_state(MODULE_STATE_READY);
+	}
+}
+
+static void start_loaging_thread()
+{
+	if (atomic_get(&thread_active)) {
+		return;
+	}
+
+	k_thread_create(&thread,thread_stack, THREAD_STACK_SIZE,
+							 (k_thread_entry_t)settings_load_thread,
+							 NULL, NULL, NULL,
+							 THREAD_PRIORITY, 0, K_NO_WAIT);
```

Comment:
indentation is off for these three lines.

Reply:
corrected